doc/workshops/financial.rst

```diff
@@ -2,4 +2,1030 @@
+
+1. The returns from applying the optimal weights of the previous year's stocks to the data for that same year (``expected_pr``).
+2. The returns from applying the optimal weights of the previous year's stocks to the following year's data (``realized_pr``).
+
+We are going to apply this function to the data in portfolio 1 and store the results in a DataFrame that we can review in the Variable Explorer.
+
+.. code-block:: python
+
+   port_1_exp_real = pd.DataFrame.from_dict(exp_real_rets(rets_1, opt_weights_1, SYMBOLS_1, start_year, end_year), orient='index')
+   port_1_exp_real.columns = ["expected", "realized"]
+
+The expected column shows the predicted values if the optimal portfolio composition had been used. The realized column, on the other hand, shows the actual profits that would have been obtained with those weights. It can be seen that there are notable differences in some years. Let's look at this in a plot.
+
+.. code-block:: python
+
+   port_1_exp_real.plot(kind="bar", figsize=(16, 10),title="Expected vs. realized Portfolio Returns")
+
+.. image:: images/workshop-1/expected-vs-realized-portfolio-1-returns.png
+   :alt: Portfolio 1 expected and realized returns comparison
+
+The most notable differences are seen in 2014 and 2016. In those years, the previous year's optimal portfolio weights (in blue in the above plot) are a lousy indicator of the following year's stock performance (in red). In 2013, our model estimated that Google and Netflix were excellent investments. But by 2014 their share price did not grow if we compare the price at the beginning and at the end of that year. And something similar happened in 2016, in which Amazon's growing trend of the previous year diminished, and the value of Microsoft and Apple shares grew.
+
+Let's summarize these numbers.
+
+.. code-block:: python
+
+   port_1_exp_real.mean()
+
+   # expected    0.408009
+   # realized    0.199700
+   # dtype: float64
+
+Our optimal weight model offered us a profit of around 40%, but the profit we would have obtained, due to real market fluctuations, would have been almost 20%. Not bad, but the mean-variance portfolio model we have applied for annual calculations is not very accurate, is it?
+
+And this result is less encouraging if we calculate the correlations between expected and realized profits.
+
+.. code-block:: python
+
+   port_1_exp_real[["expected", "realized"]].corr()
+
+   #           expected  realized
+   # expected  1.000000 -0.324053
+   # realized -0.324053  1.000000
+
+As we can see, the correlations are negative, which warns us that we should be cautious when using this type of modeling.
+
+
+
+================
+Second portfolio
+================
+
+We are now going to apply all the previous code with a portfolio of a different nature. Let's assume that instead of technology companies, we are now interested in pharmaceuticals. We will build a portfolio with stocks of Pfizer, Astra Zeneca, Johnson N Johnson.
```

Comment:
I believe this should read
```suggestion
We are now going to apply all the previous code with a portfolio of a different nature. Let's assume that instead of technology companies, we are now interested in pharmaceuticals. We will build a portfolio with stocks of Pfizer, Astra Zeneca, Johnson & Johnson.
```